draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2667,49 +2760,49 @@ uCDN-A                         dCDN-B                         dCDN-C
 
    <t hangText="Name: staleresourcetime">
    <vspace blankLines="1"/>
-   Description: The length of time for which dCDN guarantees
-         to keep a completed trigger resource. After this time,
-         dCDN SHOULD delete the trigger resource and all
-         references to it from the collection.
+   Description: The length of time for which dCDN guarantees to keep a completed trigger resource. 
+   After this time, dCDN SHOULD delete the trigger resource and all references to it from the collection.
    <vspace blankLines="1"/>
-   Value: A JSON number, which must be a positive integer,
-         representing time in seconds.
+   Value: A JSON number, which must be a positive integer, representing time in seconds.
    <vspace blankLines="1"/>
-   Mandatory: Yes, in the collection of all triggers if dCDN deletes stale entries. If the property
-        is present in the filtered collection representation, it MUST have the same
-           value as in the collection of all triggers.
+   Mandatory: Yes, in the collection of all triggers if dCDN deletes stale entries. 
+   If the property is present in the filtered collection representation, it MUST have the same
+   value as in the collection of all triggers.
    </t>
 
    <t hangText="Name: coll-state">
    <vspace blankLines="1"/>
-   Description: Array of all <xref target="filtered-collection-state-link"/> objects.
+   Description: Array of <xref target="filtered-collection-representation"/> objects.
    <vspace blankLines="1"/>
-   Value: An array of JSON-encoded FilteredCollectionStatusLink objects.
+   Value: An array of JSON-encoded filtered collection representation objects, with one entry for each 
+   trigger state, as specified in <xref target="trigger-state"/>.
    <vspace blankLines="1"/>
-   Mandatory: Mandatory in the collection of all triggers, if dCDN
-         implements the filtered collections. Otherwise, optional.
+   Mandatory: Mandatory in the trigger collection resources, if dCDN implements the per-state filtered representations 
+   of the triggers collection. Otherwise, optional.
    </t>
 
    <t hangText="Name: coll-label">
    <vspace blankLines="1"/>
-   Description: Array of all <xref target="filtered-collection-label-link"/> objects.
+   Description: Array of <xref target="filtered-collection-representation"/> objects.
    <vspace blankLines="1"/>
-   Value: An array of JSON-encoded FilteredCollectionLabelLink objects.
+   Value: An array of JSON-encoded filtered collection representation objects, with one entry for each unique label
+   key-value, as specified by uCDN, using "labels" attribtue of trigger resource (see <xref target="ci-trigger-resource"/>
```

Comment:
s/attribtue/attribute/
Also, maybe: using the "labels" attribute of a trigger resource